lib/installed_clients/AssemblyUtilClient.py

```diff
@@ -6,15 +6,15 @@
 #
 ############################################################
 
-from __future__ import print_function
+
```

Comment:
All alterations in `lib/installed_clients/` were made by running the python 2to3 script to update to py3.